play-app/src/Components/Cartcard/Cartcard.jsx

```diff
@@ -0,0 +1,78 @@
+import "./Cartcard.css"
+import {GoPlus} from "react-icons/go";
+import {BiMinus} from "react-icons/bi";
+import { useCart } from "../../Context/CartCont";
+import axios from "axios";
+import { itemWishlist } from "../../Utils/CartUtils/itemWishlist";
+
+const Cartcard = ({product}) => {
+    const {_id,image,title,newPrice,previousPrice,discount,qty} = product;
+    const {cartState,cartDispatch} = useCart();
+    const itemInYourWishlist = itemWishlist(_id,cartState.wishList );
+
+   const deleteFromCart= async(_id)=>{
+     {
+         try {
+            const cartResponse = await axios.delete(`api/user/wishlist/${_id}`,{
+                headers : {
+                    authorization: process.env.REACT_APP_ENCODED_TOKEN
+                   }
+             })
+            cartDispatch({type: 'remove-from-cart',payload: _id})
+         } catch (error) {
+             console.log(error.cartResponse)
+         }
+     }
+   }
+
+   const moveToWishlist=(product)=>{
+    cartDispatch({type: 'remove-from-cart',payload: product._id})
+    if(!itemInYourWishlist){
+    cartDispatch({type: "add-to-wishlist", payload: product})
```

Comment:
Suggestion: types should be in full caps so to decrease error chances.

Reply:
will do that